### DISCUSSION TITLE
Marking as un-watched gets re-set back to watched

### DISCUSSION THREAD

**Discussion by YorkshiremanJim** [Taxel/PlexTraktSync#1530] [General]
Is there a way to un-watch? 

For example I have things in plex that for whatever reason are marked as watched... (could be I fell asleep and woke up 2 episodes later lol) 

If I mark the episode as unplayed in plex, it gets updated back to played when it syncs again, same if you do it in trakt..  
The only way to get it to work is to do it in both which isn't the end of the world,  but would be nice to be automatic.

**Comment by simonc56** [comment]
Looks like https://github.com/Taxel/PlexTraktSync/issues/1080

**Reply by YorkshiremanJim** [reply]
ahh thank you I looked through discussions but didn't spot that!!  Seems an awkward issue lol thank you....

**Comment by YorkshiremanJim** [comment]
already a raised discussion
